test/junit/scala/tools/nsc/backend/jvm/opt/ScalaInlineInfoTest.scala

```diff
@@ -75,7 +75,7 @@ class ScalaInlineInfoTest extends BytecodeTesting {
       false, // final class
       None, // not a sam
       Map(
-        ("O()LT$O$;",                                                 MethodInlineInfo(false,false,false)),
+        ("O()LT$O$;",                                                 MethodInlineInfo(false ,false,false)),
```

Comment:
this shows scala-dev#216 was already fixed (should also fix the whitespace error)